pull_request_guidelines.md

```diff
@@ -0,0 +1,78 @@
+## Why do we do pull requests?
+
+* Catching and reducing technical debt
+* Catching mistakes
+* Improving development skills
+* Familiarity with the whole codebase
+* Maintaining consistency
+
+## Pull Request Software
+
+* Reviewboard vs. Github
+
+## Creating Pull Requests
+
+* Pull request granularity (how big should my pull requests be?)
+ * Try to keep them under 500 lines.
+* Pull request “single responsibility principle”
+ * Pull requests should have one focus, for example: Adding a new model. Or updating an API.
+ * This will also help keep them short.
+* Pull request description
+ * You should provide bullet points of the changes you are making.
+ * It is an opportunity to offer up any explanation for why you want to make these changes.
+ * Github allows you to use markdown in here, so format it well.
+* When should screenshots be included?
+ * This is usually not necessary in creating a request.
+ * Maybe if the fix you are making is impossible to explain in words?
+* [Git process, feature branches](git_workflow_quick_reference.md)
+* Assigning people vs. posting in slack
+ * When you're on a team, or working with a another specific dev, it's best to directly assign them.
+ * When the PR is more general, post in slack to get someone else's eyes on it.
```

Comment:
FWIW, backend team does "WIP" (work-in-progress) and "RFR" (ready-for-review) PRs.  Once we've started on a feature, we put up a "WIP" PR so that others can see what we're doing.  Once it's complete, we rename it with the prefix "RFR".

It's also nice to reference the Jira ticket number in the PR name or description, if there is one.